docs/integration.rst

```diff
@@ -65,6 +65,12 @@ Your reverse proxy (ex: nginx) should be configured as follow:
           }
       }
 
+To ensure that Airflow generates URLs with the correct scheme when
+running behind a TLS-terminating proxy, you should configure the proxy
+to set the `X-Forwarded-Proto` header, and enable the `ProxyFix`
+middleware in your `airflow.cfg`::
+
+    enable_proxy_fix = True
```

Comment:
Can we add a note here saying only enable this when you do have a trusted proxy in front of Airflow (i.e. nginx, AWS ELB etc.)

Reply:
Updated.